apps/evans/components/pages/home/HeroBlock.tsx

```diff
@@ -13,12 +14,16 @@ const HeroBlock: React.FC = () => {
       : `[background:linear-gradient(rgba(195,142,97,0.5),rgba(195,142,97,0.5))]`;
   return (
     <section className="HeroBlock flex w-full">
-      <BackgroundImage
-        priority
-        src={TreeRootsImage}
-        className="md:w-2/3 xl:w-3/5"
-        fit="cover"
-      >
+      <Head>
+        <link
+          rel="preload"
+          as="image"
+          imageSrcSet={`${TreeRootsSm.src} 700w, ${TreeRootsLg.src} 1200w`}
```

Comment:
this is the most important part of the eager optimization stuff i learned, which can be done with the app router too, a couple links you might want to look at:

- https://nextjs.org/docs/app/api-reference/functions/generate-metadata#resource-hints
- https://html.spec.whatwg.org/multipage/semantics.html#attr-link-imagesrcset
- https://github.com/vercel/next.js/blob/c6a061a88ecee83c60c1e2bcd4642fa6bd8f3983/packages/next/src/client/image-component.tsx#L323